frontend/src/static/js/v_summary.js

```diff
@@ -537,10 +533,8 @@ window.vSummary = {
 
     // updating filters programically //
     resetDateRange() {
-      // setting both dates to empty string will trigger their respective
-      // watchers to reset their values and remove their hashes in the url
-      this.tmpFilterSinceDate = '';
-      this.tmpFilterUntilDate = '';
+      this.tmpFilterSinceDate = this.minDate;
+      this.tmpFilterUntilDate = this.maxDate;
```

Comment:
Let this be the original code, that way you should be able to remove the guard clause, which i highlighted before. Since the double loading is gone.

Reply:
Sorry, I wasn't really clear with the reasoning behind separating the input variable and display variable.

Now I understand the reason. Have updated the code.